src/main/scala/chisel3/stage/ChiselStage.scala

```diff
@@ -172,6 +172,23 @@ object ChiselStage {
       .get
   }
 
+  def convert(chirrtl: cir.Circuit): fir.Circuit = {
```

Comment:
Scaladoc?

Reply:
Yes ScalaDoc that also discourages using this